_posts/2016-03-25-hello-squabble.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "How we added comments to our Jekyll site"
+date: 2016-03-25
+author: Kosta Harlan
+tags: jekyll
+summary: |
+  A peek under the hood at our website's comment app, Squabble.
+---
+[Avid readers of our blog](/2015/04/01/building-our-site.html) know that it's built on the amazing and lovely open-source static site generator known as [Jekyll](http://jekyllrb.com/).
+
+Jekyll is a great solution for a site like ours because we get an always-on[^1], super fast, database-free site that's also capable of many of the things you'd typically associate with a database-driven site: tagging, content organization, advanced templating, etc. And best of all it's one less database-driven application for us to maintain, keep up to date with the latest security patches, and so on.
+
+However, as much as we like static HTML, we also care about you, dear reader, and we wanted to provide an easy way for you all to comment and contribute to our site's content.
+
+## How can I add comments to a static website?
+
+Most people solve this problem in Jekyll by outsourcing the comment hosting to [Disqus](https://disqus.com/). The way this works is: you embed some JavaScript in our template, and that JavaScript loads an interface to the Disqus commenting platform. Users leave a comment via the Disqus, and on subsequent page loads, Disqus finds any comments that are added for a particular URL and displays them to the end user.
+
+Disqus works very well, but there are significant privacy concerns associated with the service as they track your activity while you browse sites across the web. (This is somewhat mitigated by this [semi-hidden setting](https://help.disqus.com/customer/portal/articles/1657951) that lets you opt out of some of their tracking, but I'm willing to venture that the vast majority of Disqus users don't know about this.) It's also not especially attractive, and the pre-built styling associated with Disqus doesn't mesh with the rest of our site.
+
+## Creating an alternative
```

Comment:
One other reason worth mentioning is to keep current with popular frameworks that are the right fit for the job. This helps others see the why in a way that reflects well on us.